src/app/hotkeys/HotkeysCheatSheet.tsx

```diff
@@ -1,26 +1,30 @@
 import KeyHelp from 'app/dim-ui/KeyHelp';
 import { t } from 'app/i18next-t';
+import { Observable } from 'app/utils/observable';
 import _ from 'lodash';
-import React, { useState } from 'react';
+import React, { memo } from 'react';
+import { useSubscription } from 'use-subscription';
 import GlobalHotkeys from './GlobalHotkeys';
-import hotkeys from './hotkeys';
 import styles from './HotkeysCheatSheet.m.scss';
+import { getAllHotkeys } from './hotkeys';
 import { useHotkey } from './useHotkey';
 
-export default function HotkeysCheatSheet() {
-  const [visible, setVisible] = useState(false);
+export const showCheatSheet$ = new Observable(false);
 
-  const toggle = () => setVisible((visible) => !visible);
+export default memo(function HotkeysCheatSheet() {
+  const visible = useSubscription(showCheatSheet$);
 
-  const hide = () => setVisible(false);
+  const toggle = () => showCheatSheet$.next(!visible);
 
-  useHotkey('?', t('Hotkey.ShowHotkeys'), toggle);
+  const hide = () => showCheatSheet$.next(false);
+
+  useHotkey('mod+/', t('Hotkey.ShowHotkeys'), toggle);
```

Comment:
This does not work correctly with non-QWERTY keyboards. With a [German QWERTZ keyboard layout](https://en.wikipedia.org/wiki/German_keyboard_layout#/media/File:KB_Germany.svg), previously I was able to do it by pressing Shift and the [?ß\\] button, now it's `Ctrl+#`, but the hotkey page itself says `Ctrl /`.

There's an FAQ entry on the About page that still mentions `?`.

Reply:
Interesting! What is Discord/Slack's shortcut?

Reply:
With QWERTZ `Ctrl+#` works for Discord, Slack uses `Ctrl+ö`.

Reply:
I went back to ? as the keybinding for now then.